mem/mem_linux.go

```diff
@@ -43,7 +48,9 @@ func VirtualMemory() (*VirtualMemoryStat, error) {
 			ret.Inactive = t * 1024
 		}
 	}
-	ret.Available = ret.Free + ret.Buffers + ret.Cached
+	if !memavail {
```

Comment:
Could you change condition to opposite such as `if memavail` ? I think it makes easier to understand. After that, I will merge this. Thanks!